charts/mirrorbits-parent/tests/custom_values_ingress_test.yaml

```
@@ -174,12 +181,13 @@ tests:
     set:
       mirrorbits-lite:
         enabled: false
-      ingress:
-        hosts:
-          - host: company.org
-            paths:
-              - path: /
-                backendService: mirrorbits
+      global:
+        ingress:
+          hosts:
+            - host: company.org
+              paths:
+                - path: /
+                  backendService: mirrorbits
```

```suggestion
                  backendService: mirrorbits-lite
```
(Depends on the acceptance or not of the first suggestions above)

---

👎 I believe this value must not depend on the name of the subchart, hence the choice of `mirrorbits` as a name here.